assets/src/infrastructure/ui/display/LoadingIndicator.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ISpinnerProps } from '@chakra-ui/core';
+
+import Spinner from './Spinner';
+
+const LoadingIndicator: React.FC<ISpinnerProps> = (props) => {
+	return <Spinner {...props} size='lg' />;
+};
```

Comment:
move this back into the app layer where it can have some sensible defaults applied, like setting a default text notice to "loading..."

The `Spinner` component you are importing here is the adapter, everything else that needs a spinner can go into the app layer and import it. Don't make duplicate adapters.

Reply:
applied "loading..." as label for screen readers, since `Spinner` does not accept `children` prop as can be seen here:

![image](https://user-images.githubusercontent.com/1477453/78984307-9ebb2b80-7b2e-11ea-8600-04fb20df8848.png)